sdk/formrecognizer/Azure.AI.FormRecognizer/src/DocumentField.cs

```diff
@@ -311,5 +326,23 @@ public AddressValue AsAddress()
 
             return ValueAddress;
         }
+
+        private object InternalValue => ValueType switch
+        {
+            DocumentFieldType.Address => AsAddress(),
+            DocumentFieldType.CountryRegion => AsCountryRegion(),
+            DocumentFieldType.Currency => AsCurrency(),
+            DocumentFieldType.Date => AsDate(),
+            DocumentFieldType.Dictionary => AsDictionary(),
+            DocumentFieldType.Double => AsDouble(),
+            DocumentFieldType.Int64 => AsInt64(),
+            DocumentFieldType.List => AsList(),
+            DocumentFieldType.PhoneNumber => AsPhoneNumber(),
+            DocumentFieldType.SelectionMark => AsSelectionMarkState(),
+            DocumentFieldType.Signature => AsSignatureType(),
+            DocumentFieldType.String => AsString(),
+            DocumentFieldType.Time => AsTime(),
+            _ => null
```

Comment:
If we remove this final case, will the compiler verify exhaustiveness for us?

Reply:
Yes. It gives us a warning (CS8524) if we remove it:
> The switch expression does not handle some values of its input type (it is not exhaustive) involving an unnamed enum value. For example, the pattern '(Azure.AI.FormRecognizer.DocumentAnalysis.DocumentFieldType)13' is not covered.